semgrep_output_v1.atd

```diff
@@ -228,14 +235,30 @@ type core_error_kind
   | OutOfMemoryDuringInterfile <json name="OOM during interfile analysis">
   (* since semgrep 0.97 *)
   | PartialParsing of location list (* list of skipped tokens *)
+  (* since semgrep 1.37.0 *)
+  | IncompatibleRule of incompatible_rule
 ] <ocaml repr="classic">
 
+type incompatible_rule
+  <ocaml attr="deriving show">
+  <python decorator="dataclass(frozen=True)"> = {
+  rule_id: string;
+  this_version: string;
+  ?min_version: string option;
```

Comment:
There is also a version type defined in this file you can use instead of string.
It's an alias for a string, but we could change that in the futur with more structure (e.g., major, minor, patch)